web/src/components/modals/remove-pep-from-pop.tsx

```diff
@@ -73,15 +88,20 @@ export const RemovePEPFromPOPModal: FC<Props> = ({
       <Modal.Footer>
         <button
           onClick={() => {
-            submit(
-              currentPeps.filter((pep) => pep.sample_name !== `${namespaceToRemove}/${projectToRemove}:${tagToRemove}`),
-              {
-                onSuccess: () => {
-                  onSuccess();
-                  onHide();
-                },
-              },
-            );
+            submit({
```

Comment:
Do we want to keep in the `onSuccess` and `onHide`? Behavior?

Reply:
Fixed